components/search_results/search_results.tsx

```diff
@@ -39,6 +39,12 @@ import './search_results.scss';
 
 const GET_MORE_BUFFER = 30;
 
+const scrollbarStyles = {
```

Comment:
@kelderek Instead of defining styles, would be better if define a class and use that

Reply:
Ok, I will give it a shot.  I originally did it this way because that is how it was done in post_list_virtualized.tsx

Reply:
done